source/bson-decimal128/decimal128.md

```diff
@@ -0,0 +1,358 @@
+# BSON Decimal128 Type Handling in Drivers
+
+- Status: Accepted
+- Minimum Server Version: 3.4
+
```

Comment:
NOTE: `Contents` section gone.

Reply:
Yes, there is no equivalent table of contents directive, we're relying on the GitHub and Pine renderers to provide them.